cow_amm/uni_v2_amm_lp_and_tvl_4047194

```diff
@@ -0,0 +1,94 @@
+WITH 
+cow_amm_pool as (
+select blockchain, address, token_1_address, token_2_address
+from query_3959044 q
+where address={{cow_amm_pool}}
+)
+
+, uniswap_pool as 
+    (select output_0 as pool from uniswap_v2_ethereum.Factory_call_getPair
+    where ((_0 in (select token_1_address from cow_amm_pool) and _1 in (select token_2_address from cow_amm_pool) )
+    or (_0 in (select token_2_address from cow_amm_pool) and _1 in (select token_1_address from cow_amm_pool)))
+    and not(output_0 = 0x0000000000000000000000000000000000000000)
+    limit 1
+    )
+    
+, get_lp_balance as (
+select date(evt_block_time) as day, 
+sum(case when "from" = 0x0000000000000000000000000000000000000000 THEN (value/1e18) ELSE -(value/1e18) END) as lp_supply
+from erc20_ethereum.evt_transfer
+where contract_address in (select pool from uniswap_pool)
+and ("from" = 0x0000000000000000000000000000000000000000 or "to" = 0x0000000000000000000000000000000000000000)
+group by 1
+)
+
+, total_lp as (
+select day, lp_supply, sum(lp_supply) over (order by day) as total_lp
+from get_lp_balance
+)
+
+, pair_sync_with_counts AS (
+    SELECT *
+        ,ROW_NUMBER() OVER (PARTITION BY date_trunc('day', evt_block_time) ORDER BY evt_block_time ASC ) AS sync_count_in_day
+    FROM uniswap_v2_ethereum.Pair_evt_Sync
+    WHERE contract_address in (select pool from uniswap_pool)
+    ORDER BY evt_block_time DESC
+)
+,pair_sync_last_in_day AS (
+    SELECT *, MAX(sync_count_in_day) OVER (PARTITION BY date_trunc('day', evt_block_time)) AS max_sync_count_in_day
+    FROM pair_sync_with_counts
+)
+,v2_pair_data AS (
+    SELECT
+        pc.*
+        ,t0.symbol AS t0_symbol
+        ,t1.symbol AS t1_symbol
+        ,t0.decimals AS t0_decimals
+        ,t1.decimals AS t1_decimals
+    FROM uniswap_v2_ethereum.Factory_evt_PairCreated AS pc
+    LEFT JOIN tokens.erc20 AS t0 ON t0.contract_address = pc.token0 AND t0.blockchain in (select blockchain from cow_amm_pool)
+    LEFT JOIN tokens.erc20 AS t1 ON t1.contract_address = pc.token1 AND t1.blockchain in (select blockchain from cow_amm_pool)
+    WHERE pc.pair in (select pool from uniswap_pool)
+)
+,token_address_list AS (
+    SELECT token0 AS token_address FROM v2_pair_data
+    UNION ALL
+    SELECT token1 AS token_address FROM v2_pair_data
+)
+,v2_reserves_by_day AS (
```

Comment:
to compute this, we could also do something similar to what you do for LP tokens, i.e., track the transfers in and out of the pool, right?

I wonder also whether there are cases of random transfers to the pool that are not proper liquidity injections (not sure why anyone would do that though), and whether in that case there is no Sync() event emitted.